src/user/USER_BASE_URL.ts

```diff
@@ -1,2 +1,4 @@
 export const USER_BASE_URL =
   "https://m.np.playstation.com/api/userProfile/v1/internal/users";
+export const GRAPHQL_BASE_URL = 
+  "https://web.np.playstation.com/api/graphql/v1/op";
```

Comment:
I am thinking this (and the new stuff) should live in its own module, perhaps _src/graphql_?

Then this file can be exported from a _GRAPHQL_BASE_URL.ts_ file.